app/Main.hs

```diff
@@ -296,6 +337,35 @@ configParser =
                 help "End line of the region to format (inclusive)"
               ]
         )
+    <*> ( FixityConfig
```

Comment:
Let's do this PR without these options and then consider what to do with them in a separate PR. I feel like already there is going to be a lot of changes and discussions.

Reply:
Should we still detect cabal dependencies in this PR then?

Reply:
Yes, like I described in the comment above. When `--cabal` is passed, get the list of direct dependencies from there, otherwise use a heuristic. These options are useful and we will have them in some form later, but I want to us to first settle on and merge a simpler version.

Reply:
I saved the current state of this branch to https://github.com/tweag/ormolu/tree/tbagrel1/fixity-with-config, so we could easily integrate the more granular fixity config later